govalid_test.go

```diff
@@ -296,19 +296,19 @@ func TestValidateCustomRule(t *testing.T) {
 }
 
 func vioMustInclude(t *testing.T, val any, msgs ...string) {
-	vio, err := govalid.Validate(val)
-	if err != nil {
-		t.Fatalf("expected nil err; found %s", err)
+	err := govalid.Validate(val)
+	if err == nil {
+		t.Fatalf("expected non nil err; found nil")
 	}
 	for _, msg := range msgs {
-		if !strings.Contains(vio, msg) {
-			t.Fatalf("expected vio to include %s; got %s", msg, vio)
+		if !strings.Contains(err.Error(), msg) {
+			t.Fatalf("expected vio to include %s; got %s", msg, err.Error())
```

Comment:
```suggestion
			t.Fatalf("expected vio to include %s; got %s", msg, err)
```

Also there is no vio, right?
```suggestion
			t.Fatalf("expected to include %s; got %s", msg, err)
```